js/src/wp-data/actions.js

```diff
@@ -0,0 +1,49 @@
+export const PREFIX = "WPSEO_WP_DATA_";
+
+export const SET_TERMS      = `${ PREFIX }SET_TERMS`;
+export const SET_TAXONOMIES = `${ PREFIX }SET_TAXONOMIES`;
+export const FETCH_FROM_API = `${ PREFIX }FETCH_FROM_API`;
+
+/**
+ * Update a list of terms in the store for a given taxonomy.
+ *
+ * @param {string} taxonomy Taxonomy slug.
+ * @param {array}  terms    List of terms.
```

Comment:
I think the JSDoc array type is with a capital. But it is preferred to specify the type it contains. In this case objects, e.g: `Object[]`.

Please also look beyond this, because you do this consistent throughout.